ResearchKit/Common/ORKInstructionStepViewController.m

```diff
@@ -60,6 +60,7 @@ - (void)stepDidChange {
         self.stepView.continueSkipContainer.continueButtonItem = self.continueButtonItem;
         self.stepView.headerView.learnMoreButtonItem = self.learnMoreButtonItem;
         self.stepView.continueSkipContainer.continueEnabled = YES;
+        self.stepView.continueSkipContainer.hidden = self.readOnlyMode;
         
         self.stepView.instructionStep = [self instructionStep];
     }
```

Comment:
Sorry, just realized that my comment was wrong. The current implementation excludes instruction steps from embedded review steps. I will happily change that :-)